pkg/predicate/predicate.go

```diff
@@ -23,7 +23,6 @@ import (
 	coordinationv1 "k8s.io/api/coordination/v1"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/utils/pointer"
```

Comment:
Can we move this entire package inside controllers? predicate.go is only used inside controllers and nowhere else.

Reply:
Sure, I'll do that